app/scripts/analysis/tools/statisitic/Statistic.module.coffee

```diff
@@ -0,0 +1,31 @@
+'use strict'
```

Comment:
- typo in the name of the directory
- `statistic` is a poor choice of name, because it's too generic, maybe something like StatsTests?